component/libfuse/libfuse.go

```diff
@@ -158,19 +158,19 @@ func (lf *Libfuse) Validate(opt *LibfuseOptions) error {
 		}
 	}
 
-	if config.IsSet(compName + ".entry-expiration") {
+	if config.IsSet(compName + ".entry-expiration-sec") {
```

Comment:
Here we check for IsSet, cant we use the same model in all components with respect to timeouts. Only if its set then use the configured value otherwise use default. File-cache/Attr-Cache/Stream/AzStorage all have some sort of timeout related configuration.

Reply:
I can change this.

Reply:
These fields are also read from CLI and config file. Issue here shall be same as that of attr-cache then.

Reply:
I dont think this is an actual issue. I think I was testing the wrong thing yesterday